### ISSUE TITLE
Suggestion: Villager List / Population

### ISSUE THREAD

**Issue by Geex204** [TangoTek/TekTopia-Community#273]
At some point i feel like it gets difficult keeping track of every villager when you town start growing faster and faster.
So it would be nice to have some sort of list somewhere in the town hall or a craftable item for the player that lists every villager with name and profession (and maybe even thier skilllevel in thier current profession). Also a note for how many beds (used and available) would be really nice.

**Comment by JohnyDL** [comment]
Yeah tell me about it some ledger would be useful instead of.... well take a look at how I've been tracking my villagers with (hundreds) of pictures in #213

**Comment by Magleose** [comment]
I was thinking the same thing. I thought maybe clicking on the town hall icon could open a GUI that would give you town stats or something like that, detailing # of villagers, professions, trade stats etc.

**Comment by JohnyDL** [comment]
I kind of like the idea of clicking on the Architect or Tradesman with a book and quill and getting them to "write" a book with stats, the book is then fixed and you can refer back to it but it's not free.
The tradesman would give an overview, how many of each profession, and then details on individual villagers of note: levelled up that day or is hurt or using 50% inventory space or wasn't able to work that day
The Architect would give a building count and then detail rooms: who sleeps in each bedroom or buildings that're missing things that would make them function better

**Comment by Konyv** [comment]
You beat me to posting this. :P
I saw this topic in the comments of the Blacksmith What If Video too.
I think the below works well for this. 

**Suggestion:** Some way to easily look up Village statistics such as type and number of structures and villagers and the number of claimed and unclaimed beds. Possibly also more advanced stats like total undead and animals kills, items produced, crops harvested. 
 
Physical Form/Interface.
Some kind of either auto-updating or manual updating display. A group of unique signs in the town hall, a unique map/painting in the town hall, or a bookcase that opens a display menu in the library or town hall. 

It could also be an interface button accessed via the Tradesmen and Architect. A button on the side that changes the menu like how clicking the book in the player inventory changes the menu.

**Comment by LinaBoeckwurm** [comment]
I would also love it if there was an option to temporarily give a glow to a certain villager on the list, so you can spot where they currently are. In case a villager gets stuck but you don't know where.

**Comment by JohnyDL** [comment]
@LinaBoeckwurm Glowy villagers is #190 ;)

**Comment by crzyaznmnky** [comment]
I really like the idea of making a town roster or manifest. This would help when trying to track down villagers so I dont have to go into all of their houses and count them in the middle of the night. It will also help with planning what villager to get next.

It could be like a map where it only updates when you hold it. It could be like a book in the library, or the town hall token, or it could be its own building like a tourist action building.

**Comment by DocNox1408** [comment]
A census would be very helpful escpecaily for servers. The team im on keeps a detailed census we name each villager by profession thus we know what dies to edit the census board. we should have over 40 farmers and over 50 guards. However people are probably loading the town and villagers die without us being there to edit the board. so The us keeping track is pretty useless.

**Comment by SaltyJohnson94** [comment]
Yes Please! This is how I have it setup in a word document. An automated, in-game version of this would be amazing!

Example:

5 Villagers/ 6 Valid Beds

Farmers (2 total):
	Dungarth Welfer (Lv 20)
	Shaun Dunhall (Lv 10)
Lumberjacks (1 Total):
	Tamond Bass (Lv 25)
Ranchers (2 Total)
	Tanika Sezlux (Lv 20)
	Diederich Carnavon (Lv 25)
